features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Scopes/Scope.js

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react'
+import { Input, Icon, Checkbox, Button, Card, Tag, Form } from 'antd';
+import { Row, Col } from 'antd';
+import Api from "../../../../data/api";
+import {message} from "antd/lib/index";
```

Comment:
Better avoid using `antd` since we have migrated to material-ui framework, and for notification, we could use react notification system library hopefully with  https://github.com/wso2/product-apim/issues/1950